ProcessMaker/Events/TemplateUpdated.php

```diff
@@ -30,10 +30,10 @@ public function __construct(array $changes, array $original, bool $processType)
         $this->processType = $processType;
 
         // Get category name
-        $this->original['process_category'] = isset($original['process_category_id'])
+        $this->original['template_category'] = isset($original['process_category_id'])
```

Comment:
Please use the key '"Category"' that has the corresponding translation

Reply:
Solved